RNTester/js/ProgressBarAndroidExample.android.js

```diff
@@ -16,33 +16,31 @@ var createReactClass = require('create-react-class');
 var RNTesterBlock = require('RNTesterBlock');
 var RNTesterPage = require('RNTesterPage');
 
-var MovingBar = createReactClass({
-  displayName: 'MovingBar',
-  _intervalID: (null: ?IntervalID),
+class MovingBar extends React.Component<{}> {
```

Comment:
Can you make this an exact type so that people don't pass additional props? `{||}`

Reply:
@gugakatsi It looks like `MovingBar` just forwards the props to `ProgressBar`. So, `MovingBarProps` should probably be defined like this:

```typescript
// Get the props of ProgressBar
type ProgressBarProps = React.ElementConfig<typeof ProgressBar>

// Remove the `progress` prop from ProgressBarProp, and make the resultant props ReadOnly
type MovingBarProps = $ReadOnly<$Diff<ProgressBarProps, {
  progress: $ElementType<ProgressBarProps, 'progress'> 
}>>
```